src/gui/tray/notificationhandler.cpp

```diff
@@ -100,6 +100,17 @@ void ServerNotificationHandler::slotNotificationsReceived(const QJsonDocument &j
 
         //need to know, specially for remote_share
         a._objectType = json.value("object_type").toString();
+        
+        // server 24: notification type chat contains conversationToken/messageId in object_type
+        if (a._objectType == "chat" || a._objectType == "call") {
+            const auto objectId = json.value("object_id").toString();
+            const auto objectIdData = objectId.split("/");
+            a._talkNotification.conversationToken = objectIdData.first();
+            if (a._objectType == "chat") {
+                a._talkNotification.messageId = objectIdData.last();
```

Comment:
@camilasan Just to avoid hard to detect issues, I would add an assert here `Q_ASSERT(objectIdData.size() > 1)` and a log message `if (objectIdData().size() < 2) { qCWarning() << "..."; }`

As in case we somehow only have 1 element, then the first and last would be identical.